src/app/shared/search/search-result.component.ts

```diff
@@ -144,14 +199,16 @@ export class SearchResultComponent implements OnChanges {
     return this.activityService.isEnabled;
   }
 
-  hyphenateIfCaseReferenceOrGet(col, result): any {
-    return col.case_field_id === '[CASE_REFERENCE]' ?
-      this.caseReferencePipe.transform(result.case_fields[col.case_field_id])
-      : result.case_fields[col.case_field_id];
+  getValueFromCaseField(col, result): any {
```

Comment:
If a [CASE_REFERENCE] field is used in search results column, it will be expected with hyphens. Is this change not removing that feature?

Reply:
I have moved this change in case tool kit